apps/web/README.md

```
@@ -0,0 +1,135 @@
+| Testes            | Vitest + Testing Library (unit em `tests/unit`, integração em `tests/integration` renderizando a `routeTree` real com `createMemoryHistory`)      |
+| Observabilidade   | Sentry via `@pipo-os/observability`                                                                                                               |
+
+## Como iniciar
+
+Além do que o README da raiz pede, o frontend precisa de duas coisas:
+
+- **Token do GitHub Packages** para o design system: um classic PAT com escopo
+  `read:packages`, em `~/.npmrc`. Sem ele o `pnpm install` falha ao resolver
+  `@piposaude/design-system`.
+
+  ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)